packages/sdk-node/src/authenticateRequest.ts

```diff
@@ -1,5 +1,6 @@
 import type { Clerk, RequestState } from '@clerk/backend';
 import { constants } from '@clerk/backend';
+import { handleValueOrFn, isHttpOrHttps, isProxyUrlRelative, isValidProxyUrl } from '@clerk/shared';
```

Comment:
@nikosdouvlis is it okay to use the import of `@clerk/shared` in `@clerk/sdk-node` or will we have issues with bundling?

Reply:
I think this is safe to do, but we do need to explicitly list the dependency in the package.json of clerk-sdk-node